nebula/ui/components/VPNGuideList.qml

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import Mozilla.VPN.qmlcomponents 1.0
+import components 0.1
+
+ColumnLayout {
```

Comment:
The reason why I made this into a `ColumnLayout` is because I needed to put different margins in between element and my attempts to use `anchors.topMargin` didn't do anything and then when I tried to set `anchors.top` it BROKE EVERYTHING, literally.